data/loader.js

```diff
@@ -12,12 +12,12 @@
 
     const folderPath = (path) => path.substring(0, path.length - path.split('/').pop().length);
     let scriptPath = (typeof window.EJS_pathtodata === "string") ? window.EJS_pathtodata : folderPath((new URL(document.currentScript.src)).pathname);
-    if (!scriptPath.endsWith('/')) scriptPath+='/';
+    if (!scriptPath.endsWith('/')) scriptPath += '/';
     //console.log(scriptPath);
     function loadScript(file) {
         return new Promise(function (resolve, reject) {
             let script = document.createElement('script');
-            script.src = function() {
+            script.src = function(){
```

Comment:
here too. I'll leave these as comments for now and review again once you're ready, all functions should be formatted like this:

```js
function() {
    // 4 spaces
    run.code();
}
```

or

```js
function name() {
    // 4 spaces
    run.code();
}
```

or


```js
const name = () => {
    // 4 spaces
    run.code();
}
```

or

```js
new Promise((resolve) => {
    // 4 spaces
    run.code();
})
```